src/js/tabs/failsafe.js

```diff
@@ -267,6 +261,11 @@ TABS.failsafe.initialize = function (callback, scrollPosition) {
                 element.prop('checked', true);
                 element.change();
                 break;
+            case 2:
+                element = $('input[id="gps_rescue"]');
+                element.prop('checked', true);
```

Comment:
Just a remark: With JQuery you can chain most commands, like `element.prop('checked', true).change()`.

Reply:
True, I did it in this way to not change the way is done in the case 1 and case 2, by coherence. Do you want that I change all of them?

Reply:
Na, just saying, both ways work equally well.